pkg/api/handlers/compat/images_prune.go

```diff
@@ -8,36 +8,29 @@ import (
 	"github.com/containers/podman/v3/libpod"
 	"github.com/containers/podman/v3/pkg/api/handlers"
 	"github.com/containers/podman/v3/pkg/api/handlers/utils"
+	"github.com/containers/podman/v3/pkg/util"
 	"github.com/docker/docker/api/types"
-	"github.com/gorilla/schema"
 	"github.com/pkg/errors"
 )
 
 func PruneImages(w http.ResponseWriter, r *http.Request) {
 	var (
 		filters []string
 	)
-	decoder := r.Context().Value("decoder").(*schema.Decoder)
 	runtime := r.Context().Value("runtime").(*libpod.Runtime)
 
-	query := struct {
-		All     bool
```

Comment:
I am not really sure what this one is doing here. I feel it does nothing. I know that All is used in libpod API, but compat has no such thing. As in my opinion, it is a dead code, I decided to remove it. Please let me know what you think.

Reply:
SGTM